Makefile

```diff
@@ -3,7 +3,7 @@ override CXX = g++
 endif
 
 ifndef CROSS_COMPILE
-    processor := $(shell uname -p)
+    processor := $(shell uname -m)
```

Comment:
Can you explain the reason why you switch from `uname -p` to `uname -m`?

Reply:
The system I was is a sort of chroot(Termux) and on a chrooted in environment. everytime I tryed to compile it always gave unknown cpu architecture since I noticed the uname command I decided that was the issue. I tried uname -p and it said unknown architecture.
![Capture+_2021-02-12-19-47-59](https://user-images.githubusercontent.com/17087430/107838379-40249680-6d6b-11eb-8e22-5d4e00f516ae.png)

Reply:
And it worked